src/utils/tests/object.utils.test.ts

```diff
@@ -0,0 +1,34 @@
+import { renameKeys } from '../object.utils'
+
+const sampleMap = { oldKeyOne: 'newKeyOne', oldKeyTwo: 'newKeyTwo' }
+describe('Object utils', () => {
+  describe('renameKeys', () => {
+    test('Changes keys based on map', () => {
+      const oldKeyOne = 'someValue1'
+      const oldKeyTwo = 'someValue2'
+      const oldMap: any = { oldKeyOne, oldKeyTwo }
```

Comment:
Not a fan of the naming of these vars (both in test and actual function above) - makes it difficult to grok. Maybe `attributeMap` and `inputMap`? Suggestions welcome